chromium_src/chrome/app/BUILD.gn

```diff
@@ -8,3 +8,17 @@ group("app") {
     "//google_apis"
   ]
 }
+
+declare_args() {
```

Comment:
this should go in a separate BUILD.gn, maybe brave/components/variations/BUILD.gn ?